src/redux/categories/categoriesSlice.js

```diff
@@ -1,15 +1,17 @@
 import { createSlice } from '@reduxjs/toolkit';
 
+// const initialState = {
+//   categories: [],
+// };
+
```

Comment:
- [ ] [**OPTIONAL**] Kindly remove useless comments to keep you code readable 😄 , and clean 👍

Reply:
- Remove unwanted commented out code for best practices.